site/src/analytics-os-boards.ts

```diff
@@ -0,0 +1,119 @@
+import "@google-web-components/google-chart";
+import { GoogleChart } from "@google-web-components/google-chart";
+import {
+  css,
+  customElement,
+  html,
+  LitElement,
+  property,
+  PropertyValues,
+  query,
+} from "lit-element";
+import { AnalyticsDataCurrent } from "../../worker/src/data";
+
+const friendlyBoardName: Record<string, string> = {
+  "intel-nuc": "Intel NUC",
+  ova: "OVA",
+  "odroid-c2": "Odroid C2",
+  "odroid-c4": "Odroid C4",
+  "odroid-n2": "Odroid N2",
+  "odroid-xu4": "Odroid XU4",
+  rpi: "Raspberry Pi",
+  rpi0: "Raspberry Pi 0",
+  rpi2: "Raspberry Pi 2",
+  rpi3: "Raspberry Pi 3",
+  rpi4: "Raspberry Pi 4",
+  tinker: "Asus TinkerBoard",
+};
+
+@customElement("analytics-os-boards")
+export class AnalyticsOsBoards extends LitElement {
+  @property({ attribute: false }) public currentData?: AnalyticsDataCurrent;
+
+  @property({ type: Boolean }) public isMobile = false;
+
+  @property({ type: Boolean }) public isDarkMode = false;
+
+  @query("google-chart") private _chart?: GoogleChart;
+
+  protected firstUpdated(_changedProperties: PropertyValues) {
+    super.firstUpdated(_changedProperties);
+    window.addEventListener("resize", () => {
```

Comment:
Should we move this to connectedCallback and then undo in disconnectCallback since we can navigate to statistics